src/index.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
 
 const defaultProps = {
   elementID: 'g-recaptcha',
-  onloadCallback: undefined,
+  onloadCallback: () => null,
```

Comment:
I didn't set `onloadCallback` to be `() => {}` because of lint's rule. Maybe we can disable the rule just for `onloadCallback` or we may set to be `() => undefined` ?